adapters/handlers/rest/configure_api.go

```diff
@@ -403,9 +390,23 @@ func MakeAppState(ctx context.Context, options *swag.CommandLineOptionsGroup) *s
 			Fatal("modules didn't initialize")
 	}
 
-	// manually update schema once
-	schema := schemaManager.GetSchemaSkipAuth()
-	updateSchemaCallback(schema)
+	// manually update schema for graphQL
+	updateSchemaCallback(schemaManager.GetSchemaSkipAuth())
+
+	// TODO-RAFT START
+	go func() {
+		if err := appState.CloudService.Open(context.Background(), executor); err != nil {
+			appState.Logger.
+				WithField("action", "startup").
+				WithError(err).
+				Fatal("could not open cloud meta store")
+		}
+		// manually update schema for graphQL after RAFT init
+		updateSchemaCallback(schemaManager.GetSchemaSkipAuth())
+	}()
+
+	time.Sleep(2 * time.Second)
```

Comment:
question(non-blocking): Why is this sleep necessary ? Mostly curious as I don't understand how it helps. Is it to give graphql time after the first empty initialization to rebuild ?

Reply:
discussed, we are going to remove it because it's unnecessary